cypress/support/index.d.ts

```diff
@@ -625,26 +627,41 @@ declare namespace Cypress {
 
 
     /**
-     * Searches and selects a specific row cell of a table by using column name and the expected value of that column.
+     * Searches and selects one or multiple rows of a table by using column's `data-cy` attribute and the expected value of that column.
      *
-     * @param columnName - name of the column
-     * @param expectedValue - expected value of the column in any row
+     * @param columnAndValue - object or array of objects having 2 fields: `{ column: 'Value', value: 'P002753' }`
      * @param modal - optional, default = false - use true if the field is in a modal overlay
```

Comment:
To which field are you referring? '..use true if the **field** is in a modal overlay' ...? You mean if the table is in a modal?